TestGxReAuthWithMidSessionPoliciesRemoval failure

https://circleci.com/api/v1.1/project/github/magma/magma/32196/output/108/0?file=true&allocation-id=5f5fbb5f6af4f93b4f403242-0-build%2F1D381A3D

[magma@10.240.0.9] out: [vagrant@127.0.0.1:2201] out: --- FAIL: TestGxReAuthWithMidSessionPoliciesRemoval (8.53s)
[magma@10.240.0.9] out: [vagrant@127.0.0.1:2201] out:     gx_reauth_test.go:171: 
[magma@10.240.0.9] out: [vagrant@127.0.0.1:2201] out:         	Error Trace:	gx_reauth_test.go:171
[magma@10.240.0.9] out: [vagrant@127.0.0.1:2201] out:         	Error:      	Expected value not to be nil.
[magma@10.240.0.9] out: [vagrant@127.0.0.1:2201] out:         	Test:       	TestGxReAuthWithMidSessionPoliciesRemoval
[magma@10.240.0.9] out: [vagrant@127.0.0.1:2201] out:         	Messages:   	Policy usage record for imsi: 000078153308272 was removed
[magma@10.240.0.9] out: [vagrant@127.0.0.1:2201] out: panic: runtime error: invalid memory address or nil pointer dereference [recovered]
[magma@10.240.0.9] out: [vagrant@127.0.0.1:2201] out: 	panic: runtime error: invalid memory address or nil pointer dereference
[magma@10.240.0.9] out: [vagrant@127.0.0.1:2201] out: [signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0x11d2488]
[magma@10.240.0.9] out: [vagrant@127.0.0.1:2201] out: 
[magma@10.240.0.9] out: [vagrant@127.0.0.1:2201] out: goroutine 860 [running]:
[magma@10.240.0.9] out: [vagrant@127.0.0.1:2201] out: testing.tRunner.func1(0xc000589400)
[magma@10.240.0.9] out: [vagrant@127.0.0.1:2201] out: 	/usr/local/go/src/testing/testing.go:830 +0x392
[magma@10.240.0.9] out: [vagrant@127.0.0.1:2201] out: panic(0x13dcb40, 0x21220f0)
[magma@10.240.0.9] out: [vagrant@127.0.0.1:2201] out: 	/usr/local/go/src/runtime/panic.go:522 +0x1b5
[magma@10.240.0.9] out: [vagrant@127.0.0.1:2201] out: magma/cwf/gateway/integ_tests.TestGxReAuthWithMidSessionPoliciesRemoval(0xc000589400)
[magma@10.240.0.9] out: [vagrant@127.0.0.1:2201] out: 	/home/vagrant/magma/cwf/gateway/integ_tests/gx_reauth_test.go:172 +0x478
[magma@10.240.0.9] out: [vagrant@127.0.0.1:2201] out: testing.tRunner(0xc000589400, 0x15d6678)
[magma@10.240.0.9] out: [vagrant@127.0.0.1:2201] out: 	/usr/local/go/src/testing/testing.go:865 +0xc0
[magma@10.240.0.9] out: [vagrant@127.0.0.1:2201] out: created by testing.(*T).Run
[magma@10.240.0.9] out: [vagrant@127.0.0.1:2201] out: 	/usr/local/go/src/testing/testing.go:916 +0x35a
[magma@10.240.0.9] out: [vagrant@127.0.0.1:2201] out: exit status 2

--- a/cwf/gateway/integ_tests/gx_reauth_test.go
+++ b/cwf/gateway/integ_tests/gx_reauth_test.go
@@ -84,6 +84,8 @@ func TestGxReAuthWithMidSessionPolicyRemoval(t *testing.T) {
 	req := &cwfprotos.GenTrafficRequest{
 		Imsi:   imsi,
 		Volume: &wrappers.StringValue{Value: *swag.String("450K")},
+		Bitrate: &wrappers.StringValue{Value: *swag.String("20M")},
+		Timeout: 60,
 	}
 	_, err := tr.GenULTraffic(req)
 	assert.NoError(t, err)
@@ -158,6 +160,8 @@ func TestGxReAuthWithMidSessionPoliciesRemoval(t *testing.T) {
 	req := &cwfprotos.GenTrafficRequest{
 		Imsi:   imsi,
 		Volume: &wrappers.StringValue{Value: *swag.String("450K")},
+		Bitrate: &wrappers.StringValue{Value: *swag.String("20M")},
+		Timeout: 60,
 	}
 	_, err := tr.GenULTraffic(req)
 	assert.NoError(t, err)
@@ -230,6 +234,8 @@ func TestGxReAuthWithMidSessionPolicyInstall(t *testing.T) {
 	req := &cwfprotos.GenTrafficRequest{
 		Imsi:   imsi,
 		Volume: &wrappers.StringValue{Value: *swag.String("450K")},
+		Bitrate: &wrappers.StringValue{Value: *swag.String("20M")},
+		Timeout: 60,
 	}
 	_, err := tr.GenULTraffic(req)
 	assert.NoError(t, err)
@@ -324,6 +330,8 @@ func TestGxReAuthWithMidSessionPolicyInstallAndRemoval(t *testing.T) {
 	req := &cwfprotos.GenTrafficRequest{
 		Imsi:   imsi,
 		Volume: &wrappers.StringValue{Value: *swag.String("450K")},
+		Bitrate: &wrappers.StringValue{Value: *swag.String("20M")},
+		Timeout: 60,
 	}
 	_, err := tr.GenULTraffic(req)
 	assert.NoError(t, err)
@@ -423,6 +431,8 @@ func TestGxReAuthQuotaRefill(t *testing.T) {
 	req := &cwfprotos.GenTrafficRequest{
 		Imsi:   imsi,
 		Volume: &wrappers.StringValue{Value: *swag.String("500K")},
+		Bitrate: &wrappers.StringValue{Value: *swag.String("20M")},
+		Timeout: 60,
 	}
 	_, err := tr.GenULTraffic(req)
 	assert.NoError(t, err)

